commands/info.md

```diff
@@ -233,6 +233,10 @@ Here is the meaning of all fields in the **stats** section:
 *   `expired_time_cap_reached_count`: The count of times that active expiry cycles have stopped early
 *   `expire_cycle_cpu_milliseconds`: The cumulative amount of time spend on active expiry cycles
 *   `evicted_keys`: Number of evicted keys due to `maxmemory` limit
+*   `total_eviction_exceeded_time`:  Total time used memory is greater than `maxmemory`
```

Comment:
I think everywhere in this doc it's referred to as ```used_memory``` and not used memory, to make it clear it's a specific value.